src/integration-test/system-integration-tests.ts

```diff
@@ -280,46 +272,55 @@ describe('system-integration', () => {
           });
         });
 
+        // TODO: for unknown reasons these fail pretty much randomly, depending on the machine and luck.
+        // If the team-scoped events are skipped, then these suddenly pass.
+        // Also when using the LocalEventBus these pass, including persistence.
```

Comment:
This was actually not the case. Even with a LocalEventBus these would fail, but weirdly the retrospective events would succeed even though they shouldn't (?). Could be good to add a TODO to check that out at some point.